snaps_k8s/playbooks/k8/k8_sriov_enable.yaml

```diff
@@ -121,7 +121,7 @@
        {
          "name": "minion-cni-network",
          "type": "multus",
-         "kubeconfig": "{{ KUBERNETES_PATH }}/node-kubeconfig.yaml",
+         "kubeconfig": "/etc/kubernetes/kubelet.conf",
```

Comment:
Should still be using {{ KUBERNETES_PATH }} here rather than hardcoding like in the other playbooks

Reply:
Incorporated.